build/vars.build.mk

```diff
@@ -14,15 +13,14 @@ LOCAL_AM_CFLAGS = \
 	-DDOCDIR=\""$(docdir)"\" \
 	-DGEANYPLUGINS_DATADIR=\""$(datadir)"\" \
 	-DPKGDATADIR=\""$(pkgdatadir)"\" \
-	-DLIBDIR=\""$(libdir)"\" \
-	-DPKGLIBDIR=\""$(pkglibdir)"\"
```

Comment:
Ok, even if not used, I changed my mind and re-added PKGLIBDIR, leave it untouched for non-Windows and set it to "lib/geany-plugins" on Windows. Not sure if this is the correct path, we will see once it is used. Then it is least there and visible.